near-sdk-macros/src/core_impl/abi/abi_generator.rs

```diff
@@ -295,3 +288,220 @@ pub fn parse_rustdoc(attrs: &[Attribute]) -> Option<String> {
         Some(doc)
     }
 }
+
+// Rustfmt removes comas.
+#[rustfmt::skip]
```

Comment:
I feel that writing / updating those expanded macros is cumbersome. Is it the case or am I missing something?

In another crate that I am driving, I configured [`insta`](https://github.com/mitsuhiko/insta) to extract the code snapshots into separate files:

https://github.com/near-cli-rs/interactive-clap/blob/0af8a4aca794568f7e33f3a5d03f138ccf20bb46/interactive-clap-derive/src/tests/test_simple_struct.rs#L7-L21

https://github.com/near-cli-rs/interactive-clap/blob/master/interactive-clap-derive/src/tests/snapshots/interactive_clap_derive__tests__test_simple_struct__flag.snap

I feel those are easier to add, maintain, and review (`cargo insta test` / `cargo insta review`). What do you think about using insta? (We should not make this change in this PR, by the way!)